libvirt/tests/src/snapshot/create_manual_snapshot.py

```
@@ -0,0 +1,253 @@
+    :param params: dict, test parameters
+    :param test: test object
+    :param disk_paths: dict of disk paths {disk_name: disk_path}
+    """
+    test.log.info("TEST_STEP4: Check the test results for bitmap flags.")
+    vm_name = params.get("main_vm")
+    disk_list = ast.literal_eval(params.get("disk_list", "[]"))
+
+    check_image_info(params, test, disk_list, disk_paths)
+    test.log.info("TEST_STEP5: Check the guest status (should be paused).")
+    if not libvirt.check_vm_state(vm_name, "paused"):
+        test.fail(f"Expected VM to be paused after snapshot, but got state: {vm.state()}")
```

Ruff (0.14.11)

Undefined name `vm`

(F821)